app/models/concerns/text_sanitizer.rb

```diff
@@ -22,4 +22,16 @@ def sanitize_text_only(text)
     return nil if text.blank?
     Loofah.fragment(text).scrub!(:prune).text(encode_special_chars: false)
   end
+
+  def sanitize_keywords(kws, strict)
+    Array(kws).map { |kw| sanitize_keyword(kw, kw.length, strict) }.uniq.reject(&:blank?)
```

Comment:
Combining a bunch of logic from elsewhere.

This takes in any input, and returns an array of unique/stripped/non-blank strings.

Optionally with `strict`, it applies our lowercasing/only-allowed-chars logic.  Which is used for `episode.keyword_xid` and matching `feed.include_tags` / `feed.exclude_tags`.

Reply:
sort?

Reply:
Hrm... I kind of think we don't want to sort?  If an RSS client is displaying these, that UI places more importance on the first category.  And maybe the user wants to control which category shows up first.

I do agree with your case-insensitive-uniq comment later.

Also fwiw ... I'm considering adding a `episode.strict_sanitized_categories` column at some point.  Which applies and stores that strict sanitization logic, so we can do a more fuzzy SQL query on these.

Reply:
yeah, that makes more sense to me than forcing sort